project/jni/openssl/compile.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-if [ -z "$ARCH_LIST" ]; then
+if [ -z "$ARCH_LIST" ] | [ "$ARCH_LIST" = "all" ]; then
```

Comment:
Is this supposed to be || not | ? It's piping the output of [ command to another [ command

Reply:
oh. oversight fixed. :-D